services/frontend-service/src/ui/utils/store.test.tsx

```diff
@@ -452,3 +462,61 @@ describe('Rollout Status', () => {
         });
     });
 });
+
+describe('Test maxActions', () => {
+    type TestDataStore = {
+        name: string;
+        inputActionsLen: number;
+        expectedLen: number;
+        expectedShowError: boolean;
+    };
+
+    const testdata: TestDataStore[] = [
+        {
+            name: 'below limit',
+            inputActionsLen: 99,
+            expectedLen: 99,
+            expectedShowError: false,
+        },
+        {
+            name: 'at limit',
+            inputActionsLen: 100,
+            expectedLen: 100,
```

Comment:
Won't this fail? We are adding 100 actions which is fine but then in 
```
    if (actions.length + 1 > maxBatchActions) {
        showSnackbarError('Maximum number of actions is ' + String(maxBatchActions));
        return;
    }
```
this will fail because 101 is larger than the 100 allowed actions

Reply:
no, at that point `actions.length` is 99. The actual "adding" comes later in the function.